redbot/cogs/mod/kickban.py

```diff
@@ -334,6 +334,15 @@ def remove_processed(ids):
                         user_id=user_id
                     )
 
+        member = namedtuple("Member", "id guild")
+        async with self.settings.guild(guild).current_tempbans() as guild_tempbans:
+            for uid in guild_tempbans.copy():
+                for user_id in user_ids.copy:
+                    if not user_id:
+                        continue
+                await self.settings.member(member(uid, guild)).banned_until.clear()
```

Comment:
```suggestion
                await self.settings.member_from_id(uid, guild).banned_until.clear()
```

Reply:
You need to de-draperize this and use self.config instead of self.settings here as well.